draft-huitema-dnssd-privacyscaling.xml

```diff
@@ -88,15 +90,17 @@
 
     <abstract>
         <t>
-DNS-SD (DNS Service Discovery) normally discloses information about both the devices offering services and the devices requesting services.
-This information includes host names, network parameters, and possibly a further description of the corresponding service instance.
-Especially when mobile devices engage in DNS Service Discovery over Multicast DNS at a public hotspot,
+DNS-SD (DNS Service Discovery) normally discloses information about both the devices offering 
+services and the devices requesting services. This information includes host names, network 
+parameters, and possibly a further description of the corresponding service instance. Especially 
+when mobile devices engage in DNS Service Discovery over Multicast DNS at a public hotspot,
 a serious privacy problem arises.
 </t>
 <t>
-The draft currently progressing in the DNSSD Working Group assumes peer-to-peer pairing between the service to be discovered
-and each of its client. This has good security properties, but create scaling issues. Each server needs to publish as many
-announcements as it has paired clients. Each client needs to process all announcements from all servers present in the
+The draft currently progressing in the DNSSD Working Group assumes peer-to-peer pairing 
+between the service to be discovered and each of its client. This has good security properties, 
+but create scaling issues. Each server needs to publish as many announcements as it has paired 
+clients. Each client needs to process all announcements from all servers present in the
 network. This leads to large number of operations when each server is paired with many clients.
 </t>
```

Comment:
I wonder whether this discussion belongs in the abstract. Also, while "each client needs to process all announcements" is true today, there is a very easy fix that does not change the nature of the proposal and let each client process exactly one announcement. But maybe we should just merge this current text, and let me rewrite it later...

Reply:
Whichever you prefer!